tests/scripts/task_python_unittest.sh

```diff
@@ -31,9 +31,9 @@ if [ -z "${TVM_UNITTEST_TESTSUITE_NAME:-}" ]; then
 fi
 
 # First run minimal test on both ctypes and cython.
-run_pytest ctypes ${TVM_UNITTEST_TESTSUITE_NAME}-platform-minimal-test tests/python/all-platform-minimal-test
-run_pytest cython ${TVM_UNITTEST_TESTSUITE_NAME}-platform-minimal-test tests/python/all-platform-minimal-test
+run_pytest --parallel ctypes ${TVM_UNITTEST_TESTSUITE_NAME}-platform-minimal-test tests/python/all-platform-minimal-test
```

Comment:
Looks like this applies to everything in the unittest directory, including `test_tvm_testing_features.py`.  I don't see the [explicit ordering](https://github.com/apache/tvm/pull/8343#discussion_r660038825) that will be required for some of those tests, so we should add it.

Reply:
Confirmed this change is in place, looks good to me.